stdlib/2and3/decimal.pyi

```diff
@@ -124,8 +124,6 @@ class Decimal(object):
         @overload
         def __round__(self) -> int: ...
         @overload
-        def __round__(self, ndigits: None) -> int: ...
-        @overload
```

Comment:
In my defense, the pure Python implementation of `Decimal.__round__()` does support `None`. But it seems the C version doesn't. Good catch.